src/calibration.c

```diff
@@ -37,3 +37,18 @@ void run_angular_speed_profile(void)
 	set_target_linear_speed(0.);
 	each(10, log_angular_speed, 2000);
 }
+
+/**
+ * @brief Run profiling test for sensors distance calculation.
+ */
+void run_distances_profiling(void)
+{
+	uint32_t clock_tick_start, clock_tick_end = 0;
```

Comment:
I think you don't need to initialize the variable to zero.

I am curious to see if `checkpatch.pl` allows for multiple variables declaration in the same line. If it does nothing to say here, I am just not used to that. :blush:

Reply:
I deleted the initialization to zero, I'm just doing it by default. I usually declare multiple variables on the same declaration to do a more compact code, and I think that the checkpatch.pl allows it. But, if it is avoided of course I will separate them.